sitemedia/js/components/SearchSort.js

```diff
@@ -0,0 +1,51 @@
+import { mapState, mapActions } from 'vuex'
+
+export default Vue.component('SearchSort', {
+    template: `
+    <div class="search-sort">
+        <label>Sort By</label>
+        <sui-dropdown
+            :value="activeSort"
+            :options="options"
+            @input="changeSort($event)"
+            selection
+        />
+    </div>
+    `,
+    computed: {
+        ...mapState([
+            'activeSort',
+            'query'
+        ]),
+        options() {
+            return [
+                {
```

Comment:
Will this be pulled from / generated via the django form in future so we don't have to duplicate options?

Reply:
Yep, that's my hope. Should be able to just pass it into the component using html.